server/broadcast.go

```diff
@@ -252,8 +253,7 @@ func selectOrchestrator(n *core.LivepeerNode, params *core.StreamParameters, cou
 		bcastOS := params.OS
 		if bcastOS.IsExternal() {
 			// Give each O its own OS session to prevent front running uploads
-			pfx := fmt.Sprintf("%v/%v", params.ManifestID, core.RandomManifestID())
-			bcastOS = drivers.NodeStorage.NewSession(pfx)
+			bcastOS = drivers.NodeStorage.NewSession(tinfo.AuthToken.SessionId)
```

Comment:
I think it would be good to do
```go
			pfx := fmt.Sprintf("%v/%v", params.ManifestID, tinfo.AuthToken.SessionId)
			bcastOS = drivers.NodeStorage.NewSession(pfx)
```
that way it would be easier to reason about data stored in the broadcaster's OS

Reply:
Ah yeah agreed that this would make it easier to reason about data stored in the broadcaster's OS. Fixed in 0b6362978236ab43dd5491a7c7f359d5a303b2f3.